src/app/[locale]/(main)/page.tsx

```diff
@@ -0,0 +1,11 @@
+import { PokemonSearch } from '@/components/PokemonSearch';
+import { Header } from '@/components/Header';
+
+export default async function HomePage() {
+  return (
+    <>
+      <Header />
```

Comment:
You can move `<Header/>` to [client-root](https://github.com/novogran/RSS_react/blob/nextjs-ssr/src/app/%5Blocale%5D/%D1%81lient-root.tsx)
 to avoid duplication in main/page and in about/page